moving_nest/fv_moving_nest.F90

```diff
@@ -0,0 +1,2517 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the FV3 dynamical core.
```

Comment:
Not sure if you want to update this section in the future, maybe make it specific to moving nest dycore?

Reply:
Good catch -- I have updated this with the wording that is used in atmos_model.F90: "This file is a part of fvGFS."